### PR TITLE
Update revisions.html.erb

### LINKED ISSUE #2664: Username in wiki/revisions pointing to /people/ instead of /profile/
Hi, this is a [first-timers-only issue](https://publiclab.github.io/community-toolbox/#r=all). This means we've worked to make it more legible to folks who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**. 

If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from groups underrepresented in free and open source software!

We know that the process of creating a pull request is the biggest barrier for new contributors. This issue is for you 💝

If you have contributed before, **consider leaving this one for someone new**, and looking through our general [help wanted](https://github.com/publiclab/plots2/labels/help-wanted) issues. Thanks!

### 🤔 What you will need to know.

Nothing. This issue is meant to welcome you to Open Source :) We are happy to walk you through the process.

### 📋 Step by Step

- [ ] 🙋 **Claim this issue**: Comment below. If someone else has claimed it, ask if they've opened a pull request already and if they're stuck -- maybe you can help them solve a problem or move it along!

- [ ] 📝 **Update** the file [app/views/wiki/revisions.html.erb](https://github.com/publiclab/plots2/blob/master/app/views/wiki/revisions.html.erb) in the `plots2` repository (press the little pen Icon) and edit the line as shown below. 

Change the link at line https://github.com/publiclab/plots2/blob/master/app/views/wiki/revisions.html.erb#L26 to 
href="/profile/<%= revision.author.name %>"

- [ ] 💾 **Commit** your changes

- [ ] 🔀 **Start a Pull Request**. There are two ways how you can start a pull request:

1. If you are familiar with the terminal or would like to learn it, [here is a great tutorial](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github) on how to send a pull request using the terminal.

2. You can also [edit files directly in your browser](https://help.github.com/articles/editing-files-in-your-repository/) and open a pull request from there. 

- [ ] 🏁 **Done** Ask in comments for a review :)

### 🤔❓ Questions?

Leave a comment below!

### Is someone else already working on this?

We encourage you to link to this issue by mentioning the issue # in your pull request, so we can see if someone's already started on it. **If someone seem stuck, offer them some help!** Otherwise, [take a look at some other issues you can help with](https://publiclab.github.io/community-toolbox/#r=all). Thanks!

### PR DESCRIPTION
**Pull request by jolivadese**
Changed line 26 link to "/profile/<%= revision.author.name %>".
Fixes #2664.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [ ] code is in uniquely-named feature branch and has no merge conflicts
* [x] PR is descriptively titled
* [x] PR body includes `fixes #0000`-style reference to original issue #
* [ ] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software

We have a loose schedule of reviewing and pulling in changes every Tuesday and Friday, and publishing changes on Fridays. 

Thanks!

### CODE REVIEW

**Comment by jolivadese** [comment]
Is the reference to #2664 ok there?

**Comment by Gauravano** [comment]
It will work for now, but writing `Fixes #2664` will close the issue automatically after your PR will merge.

**Comment by jolivadese** [comment]
How should i check the second item?

**Comment by Gauravano** [comment]
it's your branch name and also there is no merge conflict with your PR so its fine go for it.

File: app/views/wiki/revisions.html.erb [hunk 1]
```diff
@@ -23,7 +23,7 @@
           <% if revision.vid == @node.latest.vid %><span class="label label-success"><%= t('wiki.revisions.current') %></span><% end %>
         </b></td>
 
-        <% if revision.author %><td><a href="/people/<%= revision.author.name %>"><%= revision.author.name %></a></td><% end %>
+        <% if revision.author %><td><a href="/profile/<%= revision.author.name %>"><%= revision.author.name %></a></td><% end %>
```
Comment (Gauravano) [inline app/views/wiki/revisions.html.erb]: Awesome work @jolivadese !!!

**Comment by Gauravano** [comment]
Thanks for completing this @jolivadese :smiley:, if you are looking for new issues to solve, you can look at https://publiclab.github.io/community-toolbox/#r=all .

**Pull request merged**